.env_sample

```diff
@@ -110,8 +110,8 @@ export PUSHER_BEAMS_KEY=
 
 # Google recaptcha
```

Comment:
```suggestion
# Google recaptcha (Optional)
```

Do you think adding this information preserves the spirit of the original while still fixing this edge-case?